sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -8529,6 +8598,24 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ErrorType : System.IEquatable<Azure.ResourceManager.Sql.Models.ErrorType>
```

Comment:
name is too generic

Reply:
Renamed to SqlInstancePoolOperationErrorType